src/Mount.cpp

```diff
@@ -2314,6 +2325,9 @@ bool Mount::isParking() const
 /////////////////////////////////
 bool Mount::isFindingHome() const
```

Comment:
mh should we maybe just protect complete autohome code with one #if instead of multiple single lines? would be easier to maintain and read

Reply:
I'm not sure how we can do that. Certain things need to take place at certain places in the code that needs to be there in either case.